helper/definitions.py

```diff
@@ -62,6 +63,8 @@ class ComponentsTriggers(str, Enum):
     RESCHEDULING_PREPARATION = 'EXTERNAL_rescheduling_preparation_phase'
     WATCH_VIDEO = 'EXTERNAL_watch_video_dialog'
     DONE_VIDEO = 'EXTERNAL_done_with_video'
+    CONTINUE_UNCOMPLETED_DIALOG = "continue_uncompleted_dialog"
+    CENTRAL_OPTIONS = "EXTERNAL_no_valid_uncompleted_dialog"

```

Comment:
Since the naming is different (i.e., not "EXTERNAL_central_options"), might be good to add a comment for that the central options are triggered when there is no valid uncompleted dialog. Otherwise looks like an error.

Reply:
Good suggestion